src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -230,11 +230,24 @@ export default {
 		disabled(newValue) {
 			this.$refs.uploadMenu.$refs.menuButton.disabled = newValue
 		},
+
+		text(newValue) {
```

Comment:
not sure if we should debounce. the current approach seems not laggy

Reply:
No, I don't think it's needed here